bot.js

```diff
@@ -84,52 +88,35 @@ client.on('message', message => {
 	}
 
 	if (message.content.substring(0,5) === `${prefix}find` && isAdmin) {
-		const personToFind = message.content.split(" ")[1]
-		if (!personToFind) {
-			message.channel.send(`No input, please use like this: !find <IGN>`)
-			return;
+		let roster = getRoster();
+		find.findCommand(message, roster);
 		}
-		const foundMember = getRoster().find(member => member.name.toLowerCase() === personToFind.toLowerCase())
-		foundMember ? message.channel.send(JSON.stringify(foundMember)) : message.channel.send(`Zakum can't find ${personToFind} on the guild roster.`)
-	}
 
 	if (message.content.substring(0,8) === `${prefix}balance` && isAdmin) {
-		message.channel.send(`rebalancing...`)
-		balance(pool, message)
+		message.channel.send(`rebalancing...`);
+		balance(pool, message);
 	}
 
 	if (message.content.substring(0,8) === `${prefix}promote` && isAdmin) {
-		if (leaders.length === 2){
-			message.channel.send(`Zakum can't do this! There are already two expedition leaders.`)
-			return;
-		}
-		const name = message.content.split(" ")[1]
-		let roster = getRoster()
-		let promoted = roster.find(member => member.name.toLowerCase() === name.toLowerCase())
-		promoted.leader = true
-		leaders.push(promoted)
-		fs.writeFile("./guilds/lt3.json", JSON.stringify({"lt3":roster}, null, 4), (err) => {
-		    if (err) {
-		        console.error(err);
-		        return;
-		    };
-		    message.channel.send(`Promoted ${name} to expedition leader!`)
-		});
+		let roster = getRoster();
+		promote.promoteCommand(message, roster, leaders);
+	}
+
+	if (message.content.substring(0,8) === `${prefix}leaders` && isAdmin) {
+		let roster = getRoster();
```

Comment:
This can be done in one line:` message.channel.send(getRoster.filter(member => member.leader).join(" ,"))`